src/numbers.js

```diff
@@ -1,45 +1,47 @@
-function add (a, b) {
-  // your code here
-}
 
-function subtract (a, b) {
-  // your code here
+  const add = (a, b) => {
+    return a + b;
+  };
+
+
+const subtract = (a, b) => {
+  return a - b;
 }
 
-function multiply (a, b) {
-  // your code here
+const multiply = (a, b) => {
+  return a * b;
 }
 
-function divide (a, b) {
-  // your code here
+const divide = (a, b) => {
+  return a / b;
 }
 
-function power (a, b) {
-  // your code here
+const power = (a, b) => {
+  return a ** b;
 }
 
-function round (a) {
-  // your code here
+const round = (a) => {
+  return Math.round(a);
 }
 
-function roundUp (a) {
-  // your code here
+const roundUp = (a) => {
+  return Math.ceil(a);
 }
 
-function roundDown (a) {
-  // your code here
+const roundDown = (a) => {
+  return Math.floor(a);
 }
 
-function absolute (a) {
-  // your code here
+const absolute = (a) => {
+  return Math.abs(a);
 }
 
-function quotient (a, b) {
-  // your code here
+const quotient = (a, b) => {
+  return parseInt(a / b);
 }
 
-function remainder (a, b) {
-  // your code here
+const remainder = (a, b) => {
+  return Math.floor(a % b);
```

Comment:
you do not need to use `Math.floor()` here. simply doing `a % b` gets you the same result